--- configure	2019-06-25 10:01:44.842826162 +0200
+++ configure.win	2019-06-25 11:24:05.051065060 +0200
@@ -276,7 +276,7 @@
 fi
 fi
 fi
-SHELL=${CONFIG_SHELL-/bin/sh}
+SHELL=${CONFIG_SHELL-sh}
 export SHELL
 # Unset more variables known to interfere with behavior of common tools.
 CLICOLOR_FORCE= GREP_OPTIONS=
@@ -485,8 +485,8 @@
       t loop
       s/-\n.*//
     ' >$as_me.lineno &&
-  chmod +x "$as_me.lineno" ||
-    { $as_echo "$as_me: error: cannot create $as_me.lineno; rerun with a POSIX shell" >&2; as_fn_exit 1; }
+#  chmod +x "$as_me.lineno" ||
+#    { $as_echo "$as_me: error: cannot create $as_me.lineno; rerun with a POSIX shell" >&2; as_fn_exit 1; }
 
   # If we had to re-execute with $CONFIG_SHELL, we're ensured to have
   # already done that, so ensure we don't try to do so again and fall
@@ -1311,11 +1311,11 @@
   --without-PACKAGE       do not use PACKAGE (same as --with-PACKAGE=no)
   --with-cvodes_include=CVODES_INCLUDE_DIR
                           the location of CVODES header files, where
-                          cvodes/cvodes.h resides (default /usr/local/include)
+                          cvodes/cvodes.h resides (default %ProgramFiles%\cvodes\include)
   --with-cvodes_libs=CVODES_LIBS_DIR
                           the location of CVODES libraries, where
                           libsundials_cvodes.so resides (default
-                          /usr/local/lib64)
+                          %ProgramFiles%\cvodes\lib)
 
 Some influential environment variables:
   CXX         C++ compiler command
@@ -2030,7 +2030,7 @@
    if test  -n "${CVODES_INCLUDE}"  ; then
       CVODES_INCLUDE="${CVODES_INCLUDE}"
    else
-      CVODES_INCLUDE="/usr/local/include"
+      CVODES_INCLUDE="%ProgramFiles%\cvodes\include"
    fi
 fi
 
@@ -2968,7 +2968,7 @@
    if test  -n "${CVODES_LIBS}"  ; then
       CVODES_LIBS="${CVODES_LIBS}"
    else
-      CVODES_LIBS="/usr/local/lib64"
+      CVODES_LIBS="%ProgramFiles%\cvodes\lib"
    fi
 fi
 
@@ -3511,7 +3511,7 @@
 ## Main body of $CONFIG_STATUS script. ##
 ## ----------------------------------- ##
 _ASEOF
-test $as_write_fail = 0 && chmod +x $CONFIG_STATUS || ac_write_fail=1
+#test $as_write_fail = 0 && chmod +x $CONFIG_STATUS || ac_write_fail=1
 
 cat >>$CONFIG_STATUS <<\_ACEOF || ac_write_fail=1
 # Save the log message, to keep $0 and so on meaningful, and to
@@ -3581,7 +3581,7 @@
 
 ac_pwd='$ac_pwd'
 srcdir='$srcdir'
-test -n "\$AWK" || AWK=awk
+test -n "\$AWK" || AWK=gawk
 _ACEOF
 
 cat >>$CONFIG_STATUS <<\_ACEOF || ac_write_fail=1
